src/applications/my-education-benefits/containers/App.jsx

```diff
@@ -2,61 +2,39 @@ import React, { useEffect } from 'react';
 import { connect } from 'react-redux';
 import { setData } from 'platform/forms-system/src/js/actions';
 import Breadcrumbs from '@department-of-veterans-affairs/component-library/Breadcrumbs';
-
 import RoutedSavableApp from 'platform/forms/save-in-progress/RoutedSavableApp';
 import formConfig from '../config/form';
-
 import { fetchPersonalInformation, fetchEligibility } from '../actions';
+import { fetchUser } from '../selectors/userDispatch';
+import { personalInfoFetchProgress } from '../selectors/personalInfoFetchInProgress';
 
 export const App = ({
-  // loggedIn,
-  // showNod,
   location,
   children,
-  // profile,
   formData,
   setFormData,
   getPersonalInfo,
   firstName,
   getEligibility,
   eligibility,
+  user,
```

Comment:
I want to check for 2 things before fetching claimant info:

- That we are logged in.
- That there is no other fetch in progress.